tmk_core/protocol/lufa/lufa.c

```diff
@@ -1083,6 +1084,10 @@ int main(void)
         MIDI_Device_USBTask(&USB_MIDI_Interface);
 #endif
 
+#ifdef VELOCIKEY_ENABLE
+    if (velocikey_enabled()) velocikey_decelerate();    
```

Comment:
Please change this to: 
```suggestion
    if (velocikey_enabled()) { velocikey_decelerate();  }
```
Per QMK Coding conventions: https://docs.qmk.fm/#/contributing?id=coding-conventions